src/modules/relayer/executeAuth.middleware.ts

```diff
@@ -64,15 +65,17 @@ export async function validateExecuteSignature(
   const universalProfile = UniversalProfile__factory.connect(address, provider);
   const keyManagerAddress = await universalProfile.owner();
 
+  const { signature, nonce, abi, validityTimestamps } = transaction;
+
   const message = ethers.utils.solidityPack(
     ["uint256", "uint256", "uint256", "uint256", "uint256", "bytes"],
     [
-      6,
+      LSP25_VERSION,
       CHAIN_ID,
-      transaction.nonce,
-      transaction.validityTimestamps || 0,
-      0,
-      transaction.abi,
+      nonce,
```

Comment:
can we keep this as transaction.nonce. I know usually we want to structure things but for developers it is easier to follow seeing it the nonce of the transactions, otherwise it can be confused with the nonce of the signing key